mw-webapp/src/component/image/Image.module.scss

```diff
@@ -3,4 +3,26 @@
   height: 100%;
   object-fit: cover;
   object-position: center;
+  transition: transform 0.3s ease;
+
+  &.zoomable {
+    cursor: pointer;
+  }
+}
+
+.zoomedOverlay {
+  position: fixed;
+  z-index: 1000;
```

Comment:
PLease look styles in other files. We using global var for such variable. Check everywhere